.github/workflows/clang-tidy-review.yml

```diff
@@ -0,0 +1,72 @@
+---
+name: clang-tidy-review
```

Comment:
Same as with the other one. Name should be human readable right? I'd prefer something like "run Clang tidy linting"